### ISSUE TITLE
Firebase Auth - Incompatible

### ISSUE THREAD

**Issue by broberts-nybble** [FirebaseExtended/flutterfire#162]
**Describe the bug**
When using the latest version, it appear to cash the entire app for no apparent reason, when downgrading it works. 

This is not an uncommon issue with the firebase libraries, please can you look into it properly and correct it, you are causing us a lot of pain.

**Comment by jayvasantjv** [comment]
I'm not facing any issues while working with latest version.

Kindly elaborate what kind of issue you're facing.

**Comment by BondarenkoStas** [comment]
@broberts-nybble Can you reproduce it on any project, or on the some exact one?

**Comment by broberts-nybble** [comment]
I was able to find the issue / conflicts:

1. Firebase auth - conflicting with firebaseml & firebase core, versions we're all on android-x, but after migration it helped
2. Firebase auth - major conflict with "local_auth" package

The documentation when it crashes / fails isn't really helpful which leads you on a mindless hunt, but this is approx. the 3rd time I've updated these libraries and each time there are issues where it crashes the app and creates instabilities.
